source/install/troubleshooting.rst

```diff
@@ -19,9 +19,40 @@ General Troubleshooting
 -----------------------
 Some of these suggestions can be done directly, and others may need consultation from your network administrator.
 
-- Take a look at the logs (``mattermost.log`` and NGINX logs) for errors.
-- You can also search the error messages online - existing solutions can often be applied.
-- Open **System Console > General > Logging** and set File Log Level to **DEBUG**. Make sure to revert to **INFO** after troubleshooting to save disk space.
+Take a Look at the Logs
+~~~~~~~~~~~~~~~~~~~~~~~
+
+This article covers accessing the numerous logs created by Mattermost applications.
+
+**Mattermost Server**
+
+1. In the *System Console* go to *Environment > Logging* and note down the path to the *File Log Directory*
+2. The server log file is called *mattermost.log* and can be opened with a standard text editor or shared directly.
+
+.. Note::
+   Make sure to verify **logging to a file** is enabled by confirming **System Console > Environment > Logging > Output logs to file** is set to **true**. Open **System Console > Environment > Logging** and set File Log Level to **DEBUG**. Make sure to revert to **INFO** after troubleshooting to save disk space. Logging can be set to output plain text or JSON here as well.
+
+If filesystem access is not possible the current server logs can be found and copied from the *System Console* under *Reporting > Server Logs*.
+
+`More <https://docs.mattermost.com/administration/config-settings.html#logging>`_
```

Comment:
This may work better as a sentence to add some context.

Reply:
like 
> You can find more on logging [here](https://docs.mattermost.com/administration/config-settings.html#logging).

?

Reply:
+1, propose adding "settings" to indicate it's about the setting specifically - `You can find more on logging settings <here>.`